spec/rubocop/cop/rspec/spec_file_path_format_spec.rb

```diff
@@ -281,4 +281,272 @@ class Foo
       RUBY
     end
   end
+
+  shared_context 'with successful require stubbing' do
+    before do
+      allow(TOPLEVEL_BINDING.receiver).to receive(:require).and_call_original
+      allow(TOPLEVEL_BINDING.receiver).to receive(:require)
+        .with('active_support/inflector').and_return(true)
+      allow(TOPLEVEL_BINDING.receiver).to receive(:require)
+        .with(cop_config['InflectorPath']).and_return(true)
+    end
+  end
+
+  shared_context 'with ActiveSupport available' do
+    before do
+      allow(described_class::ActiveSupportInflector).to receive(:available?)
+        .and_return(true)
+    end
+  end
+
+  shared_context 'when using ActiveSupport integration' do
+    around do |example|
+      described_class.reset_activesupport_cache!
+      example.run
+      described_class.reset_activesupport_cache!
+    end
+
+    before do
+      # rubocop:disable RSpec/VerifiedDoubleReference
+      inflector = instance_double('ActiveSupport::Inflector')
+      # rubocop:enable RSpec/VerifiedDoubleReference
```

Comment:
`RSpec/VerifiedDoubleReference` correctly complains that there’s no point in using `instance_double` with a String argument, because RSpec cannot verify which methods are available unless the class is loaded. And it won’t be 😅 Let’s just use a `double` instead.

```suggestion
      inflector = double('ActiveSupport::Inflector') # rubocop:disable RSpec/VerifiedDoubles
```

(Also, had the class been loaded, I think `class_double` would have been a better choice.)

Reply:
I think I had a double and it got autocorrected for another violation, at which point I disabled it. Good call, I will clean this up.

Reply:
Okay, fixed to violate `VerifiedDoubles` instead, and added back the explanatory comment inline in the specs